appverifier/src/main/java/com/android/mdl/appreader/fragment/RequestOptionsFragment.kt

```diff
@@ -140,6 +143,10 @@ class RequestOptionsFragment : Fragment() {
         } else {
             RequestOptionsFragmentDirections.toSelectTransport(requestedDocuments)
         }
+        if (findNavController().currentDestination?.id == R.id.presentationLogs) {
```

Comment:
if user is actively viewing the History fragment, "back out" before navigating to another destination